tests/recovery/recovery_test/recovery_test.go

```diff
@@ -0,0 +1,133 @@
+//go:build darwin || (linux && amd64)
+// +build darwin linux,amd64
+
+package recovery
+
+import (
+	"errors"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kopia/kopia/tests/recovery/blobmanipulator"
+	"github.com/kopia/kopia/tests/robustness/snapmeta"
+	"github.com/kopia/kopia/tests/tools/kopiarunner"
+)
+
+func TestSnapshotFix(t *testing.T) {
+	// assumption: the test is run on filesystem & not directly on object store
+	dataRepoPath := path.Join(*repoPathPrefix, dataSubPath)
+
+	baseDir := makeDir("base-dir-")
+	if baseDir == "" {
+		t.FailNow()
+	}
+
+	bm, err := blobmanipulator.NewBlobManipulator(baseDir)
+	if err != nil {
+		if errors.Is(err, kopiarunner.ErrExeVariableNotSet) {
+			log.Println("Skipping robustness tests because KOPIA_EXE is not set")
```

Comment:
Does this need `t.Skip` instead? the log message seems to imply skip.
```suggestion
			t.Skip("Skipping robustness tests because KOPIA_EXE is not set")
```